File: app/javascript/channels/game_channel.js
import consumer from "./consumer"

consumer.subscriptions.create({channel: 'GameChannel', bingo_hall_id: 1}, {
  connected() {
  //Called when the subscription is ready for use on the server
    console.log("we made it...")
  },

  disconnected() {
    // Called when the subscription has been terminated by the server
  },

  received(data) {
    // Called when there's incoming data on the websocket for this channel
  },

  toggleSquare(data) {

    $("#").onClick(function(event) {
    //if color background == white change to red
      // //if color background == red change to white
    //toggle state via applying or removing a background color

  };
  }
});

File: app/channels/game_channel.rb
class GameChannel < ApplicationCable::Channel
  def subscribed
    game = BingoHall.find(params[:bingo_hall_id])
    stream_for game
  end

  def unsubscribed
    # Any cleanup needed when channel is unsubscribed
  end

  # def receive(data)
  #   ActionCable.server.broadcast("#{params[:bingo_hall_id]}", data)
  # end
end
